prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -111,7 +111,7 @@ if [[ $SYNCMODE == 1 ]]; then
     MCH_CONFIG_KEY="MCHTracking.maxCandidates=50000;MCHTracking.maxTrackingDuration=20;"
     MFT_CONFIG_KEY+="MFTTracking.cutMultClusLow=0;MFTTracking.cutMultClusHigh=20000;"
   elif [[ $BEAMTYPE == "pp" ]]; then
-    ITS_CONFIG_KEY+="fastMultConfig.cutMultClusLow=${CUT_MULT_MIN_ITS:--1};fastMultConfig.cutMultClusHigh=${CUT_MULT_MAX_ITS:-1};fastMultConfig.cutMultVtxHigh=${CUT_MULT_VTX_ITS:--1};ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2;"
+    ITS_CONFIG_KEY+="fastMultConfig.cutMultClusLow=${CUT_MULT_MIN_ITS:--1};fastMultConfig.cutMultClusHigh=${CUT_MULT_MAX_ITS:--1};fastMultConfig.cutMultVtxHigh=${CUT_MULT_VTX_ITS:--1};ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2;"
```

Comment:
```suggestion
    ITS_CONFIG_KEY+="fastMultConfig.cutMultClusLow=${CUT_MULT_MIN_ITS:--1};fastMultConfig.cutMultClusHigh=${CUT_MULT_MAX_ITS:--1};fastMultConfig.cutMultVtxHigh=${CUT_MULT_VTX_ITS:--1};ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2;ITSCATrackerParam.TrackletsPerClusterLimit=${CUT_TRACKLETSPERCLUSTER_MAX_ITS:--1};ITSCATrackerParam.cellsPerClusterLimit=${CUT_CELLSPERCLUSTER_MAX_ITS:--1};"
```

Reply:
I guess you meant `TSCATrackerParam.trackletsPerClusterLimit=${CUT_TRACKLETSPERCLUSTER_MAX_ITS:--1};` ?
This will work, but is there a reason of keeping 2 for sync.mode while for the async it is `10^3` (together with other more relaxed settings)? I don't like that the sync with default settings will never work unless prescaled.

Reply:
> I guess you meant `TSCATrackerParam.trackletsPerClusterLimit=${CUT_TRACKLETSPERCLUSTER_MAX_ITS:--1};` ? 

Sure, there was a typo.

> I don't like that the sync with default settings will never work unless prescaled.

To be honest, I don't like it either; I'd also prefer FST to test something reasonably close to the actual use case and not a custom setup that "just works".

My point was to preserve what was used in `sync` mode for consistency, wrt what was used.

At this point, since the effect of these cuts in sync mode is to drop the reconstruction, I think we could make them as large as the `async` ones and entirely rely on `fastMultEst` for a meaningful drop of ROFs.
We then monitor the next PbPb reco and possibly fine-tune them if it turns out tracking becomes too slow.

Reply:
The prescaling in the online reco was meant just to save the CPU (both on ITS and following barrel global tracking) and so far, with the current prescaling, we never hit the threshold. In opposite, with the sync reco w/o prescaling used for the residuals extraction we were always hitting the threshold with ~10 tracklet / clusters. So, with the values I set (100) the behaviour of the online processing should not change but the non-prescaled processing should work w/o posing memory problems (in async the threshold is 1000).

Reply:
Hi @shahor02, the threshold were tuned to the pp case, so indeed for Pb-Pb they might be tight. I don't have strong opinions about how to loosen them for Pb-Pb while keeping the same defaults in pp. Mind that if the application of un-prescaled Pb-Pb sync processing is only the residual extraction we should now use the new async mode with the per vertex processing, so the threshold should not be there.

Reply:
@mpuccio but the threshold of 2 was never hit in the online processing, neither in pp nor in pbpb, with the prescaling we are applying. Does not this mean that increasing the threshold should not have any effect on the online processing?
As I wrote, I don't like the fact that current sync mode works only if prescaling is applied.

Reply:
> The prescaling in the online reco was meant just to save the CPU (both on ITS and following barrel global tracking) and so far, with the current prescaling, we never hit the threshold. In opposite, with the sync reco w/o prescaling used for the residuals extraction we were always hitting the threshold with ~10 tracklet / clusters. So, with the values I set (100) the behaviour of the online processing should not change but the non-prescaled processing should work w/o posing memory problems (in async the threshold is 1000).

Then, it is my misunderstanding of the meaning of the prescaler.
I thought that was in place to regulate the minimum amount of reconstructed tracks fed to the TPC online reco/compression.
I then advocate keeping resource management for ITS reconstruction inside the ITS reconstruction itself.
Otherwise, we might have different levels of mechanisms to configure the tracker directly and indirectly, which is not optimal and ends up in ambiguous situations like this.